mappings/net/minecraft/entity/data/TrackedDataHandlerRegistry.mapping

```diff
@@ -7,21 +7,36 @@ CLASS net/minecraft/unmapped/C_fegtripb net/minecraft/entity/data/TrackedDataHan
 	FIELD f_dwamwwzn OPTIONAL_UUID Lnet/minecraft/unmapped/C_xrzqxzbg;
 	FIELD f_hdycstqt ITEM_STACK Lnet/minecraft/unmapped/C_xrzqxzbg;
 	FIELD f_idrgfatj STRING Lnet/minecraft/unmapped/C_xrzqxzbg;
+	FIELD f_ifjlggjg OPTIONAL_GLOBAL_POSITION Lnet/minecraft/unmapped/C_xrzqxzbg;
```

Comment:
Could this be shortened to OPTIONAL_POSITION

Reply:
It's a global position type.